scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+
+The [GVS pipeline](https://github.com/broadinstitute/gatk/blob/rc-vs-483-beta-user-wdl/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) is an open-source, cloud-optimized workflow for joint calling at a large scale using the Broad GVS. The workflow takes in up to 100,000 single sample gVCF files and combines them into a variant filtering model driven by machine learning. The model is applied to the data, and a sharded joint VCF with variant calls is output.
+
+The filtering model is based on the [WARP Joint Genotyping workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.wdl) and uses the [Variant Quality Score Recalibration (VQSR)](https://gatk.broadinstitute.org/hc/en-us/articles/360035531612) technique which uses machine learning to model the technical profile of variants to flag probable artifacts.
+
+## Quickstart table
+
+The following table provides a quick overview of the GVS pipeline features:
+
+| Pipeline features | Description | Source | 
+| --- | --- | --- |
+| Overall workflow | Joint Calling with the Broad Genomic Variant Store using a machine learning-powered filter model | Code available from [GitHub](https://github.com/broadinstitute/gatk/blob/rc-vs-483-beta-user-wdl/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) |
+| Workflow language | WDL 1.0 | [openWDL](https://github.com/openwdl/wdl) |
+| Genomic reference sequence | GRCh38 (hg38) human genome primary sequence | Genome Reference Consortium [GRCh38](https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.39) |
+| Data input file format | File format in which input data is provided | [gVCF](https://gatk.broadinstitute.org/hc/en-us/articles/360035531812) |
+| Data output file formats | File formats in which outputs are provided | [VCF](https://gatk.broadinstitute.org/hc/en-us/articles/360035531692) and associated index files; TXT (manifest) |
+
+## Setup
+
+### Getting started with Terra
+
+The GVS workflow is available on Terra, a cloud-based platform for biomedical research. After registering, you can access the [GVS workspace](LINK), which contains instructions for performing joint calling with the Broad GVS using example data uploaded to the workspace. For more information on using the Terra platform, see the [Terra Support Center](https://support.terra.bio/hc/en-us).
+
+If you are new to Terra, you’ll need to [register for a Terra account](https://support.terra.bio/hc/en-us/articles/360028235911). Beta testers of the GVS will be given access to a Broad-managed billing project, but to use Terra for other projects, you’ll need to [set up a billing project](https://support.terra.bio/hc/en-us/articles/360046295092).
```

Comment:
Are we going to give them access to a Broad billing project?  That seems… risky since they could run up a massive bill in there doing other work, making mistakes, etc.  Can we use the free-credits approach here?

Reply:
This is going to be a Kylee question!